### PR TITLE
LPS-152539 Add link to instance settings in segments warnings messages

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
Add breadcrumb to Segments Disabled warning in segments list and segments editor.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-152539)

Proposed Solution
========
Update the clay:stripe with a link to instance settings if the user have permission to access instance settings. If not, it will show a text.

Steps to Verify:
----------------
1. Log in with Test user
1. In Instance settings disable segmentation
1. Go to People -> Segments and check that the warning shows a link to instance settings
1. Go to the segments editor (editing a segment or creating a new one) and check that the warning shows a link.
1. Create a user with no permissions for instance settings and repeat the previous steps. Check that the warnings do not show a link. (People -> Membership, select the user and assign roles as Site Owner)

Screenshots (optional):
-----------------------
Captura de pantalla 2022-06-23 a las 9 03 02

Captura de pantalla 2022-06-23 a las 9 02 48

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
Hey @claraizquierdo !!

Nice work!!

I have only one doubt about the AC:

_When the user does not have the proper permissions, the "segmentation disabled" alert must inform the user to contact their system admin._ 

It will be implemented in another PR?

Thanks!!

**Comment by claraizquierdo** [comment]
> Hey @claraizquierdo !!
> 
> Nice work!!
> 
> I have only one doubt about the AC:
> 
> _When the user does not have the proper permissions, the "segmentation disabled" alert must inform the user to contact their system admin._
> 
> It will be implemented in another PR?
> 
> Thanks!!

You are right Cristina! I think a prefer to fix it in this PR

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js [hunk 1]
```diff
@@ -564,8 +566,18 @@ class SegmentEdit extends Component {
 								)}
 							</strong>
 
-							{Liferay.Language.get(
-								'to-enable,-go-to-instance-settings'
+							{this.props.segmentsConfigurationURL ? (
```
Comment (beltranrengifo) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/segment_edit/SegmentEdit.es.js:569]: I'd maybe follow here the pattern of _destructuring_ the props object in line 446 and use the variable directly. Just to have all in one place...

**Review by beltranrengifo — APPROVED** [review]
hi @claraizquierdo ! Everything works properly :)
I left you a minor comment inline.

I believe the spacing of the items inside the alerts is wrong, feels unnatural, but that's another story, literally 😄 
Also, the _Warning_ text with the colon that appears in the jsp taglib is not in the original design. We talked about that [here](https://liferay.slack.com/archives/CNBG06JS3/p1655282469870799), no final decision yet 🤷

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:reevaluate:1257401_3325

**Comment by claraizquierdo** [comment]
ci:stop

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
Ey @cgoncas , looks like the test `com.liferay.segments.web.internal.portlet.action.test.EditSegmentsEntryMVCRenderCommandTest.testGetProps` is broken. Could you help me to fix it?

**Comment by cgoncas** [comment]
> Ey @cgoncas , looks like the test `com.liferay.segments.web.internal.portlet.action.test.EditSegmentsEntryMVCRenderCommandTest.testGetProps` is broken. Could you help me to fix it?

Yeahh!!!, Maybe it is related with [LPS-156828](https://issues.liferay.com/browse/LPS-156828)?

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**